zebra/rt_netlink.c

```diff
@@ -1119,9 +1119,12 @@ static void _netlink_route_build_singlepath(const char *routedesc, int bytelen,
 	mpls_lse_t out_lse[MPLS_MAX_LABELS];
 	char label_buf[256];
 	int num_labels = 0;
+	struct vrf *vrf = NULL;
```

Comment:
Can we make this unified everywhere you touch it? I mean initialize to NULL or not, but not mixed.

Reply:
Yeah, you're totally right. Mea culpa.